src/libkernel/sscp/amdgpu/integer.cpp

```diff
@@ -37,3 +37,16 @@ HIPSYCL_SSCP_BUILTIN __hipsycl_int32 __hipsycl_sscp_mul24_s32(__hipsycl_int32 a,
 HIPSYCL_SSCP_BUILTIN __hipsycl_uint32 __hipsycl_sscp_mul24_u32(__hipsycl_uint32 a, __hipsycl_uint32 b) {
   return __ockl_mul24_u32(a, b);
 }
+
+HIPSYCL_SSCP_BUILTIN __hipsycl_uint8 __hipsycl_sscp_clz_u8(__hipsycl_uint8 a){
+  return __ockl_clz_u8(a);
+}
+HIPSYCL_SSCP_BUILTIN __hipsycl_uint16 __hipsycl_sscp_clz_u16(__hipsycl_uint16 a){
+  return __ockl_clz_u16(a);
+}
```

Comment:
Why do we have u8 and u16 implmentations here? The header does not define these builtins, and the SYCL interface does not use them. If there is some benefit to being able to use the OCKL u8 and u16 functions, we should expose 8 and 16-bit versions and use them in the SYCL headers.